pyproject.toml

```diff
@@ -1,6 +1,56 @@
 [build-system]
-    build-backend = "setuptools.build_meta"
-    requires = [
-        "setuptools>=47",
-        "wheel>=0.34",
-    ]
+build-backend = "setuptools.build_meta"
+requires = [
+  "setuptools>=47",
+  "wheel>=0.34",
+]
+
+[project]
+name = "python-mimeparse"
+readme = "README.rst"
+dynamic = ["version"]
+dependencies = []
+requires-python = ">=3.8"
+description = "A module provides basic functions for parsing mime-type names and matching them against a list of media-ranges."
+license = {text = "MIT"}
```

Comment:
This doesn't include the license file. It's probably fine since mit should be cleat enough by itself

Reply:
Yes, it should be enough.

However, for `setup.cfg` parity, we do include the license file later:
```toml
[tool.setuptools]
license-files = ["LICENSE"]
```